config/tf_modules/gcp-mysql/main.tf

```diff
@@ -0,0 +1,49 @@
+resource "random_id" "key_suffix" {
+  byte_length = 8
+}
+
+resource "random_password" "root_auth" {
+  length  = 20
+  special = false
+}
+
+# TODO: add encryption key name once out of beta
```

Comment:
what does this mean?

Reply:
Disk encryption in GCP is currently in beta

Reply:
So I decided not to tangle with that and the _separate_ provider it would make us use

Reply:
Ah so is it currently encrypted or not?

Reply:
Please clarify in the comment that "disk encryption in gcp mysql is currently in beta"